pom.xml

```diff
@@ -74,6 +74,9 @@
         <!-- Mixed Reality -->
         <module>sdk/mixedreality/azure-mixedreality-authentication</module>
 
+        <!-- Models Repository -->
+        <module>sdk/modelsrepository/azure-iot-modelsrepository</module>
+
         <!-- Quantum Jobs -->
         <module>sdk/quantum/azure-quantum-jobs</module>
```

Comment:
This SDK sounds way cooler than any SDK our team owns

Reply:
:))) eyes up here !